code/modules/jobs/job_types/_job.dm

```diff
@@ -145,6 +146,9 @@
 	if(CONFIG_GET(flag/everyone_has_maint_access)) //Config has global maint access set
 		. |= list(ACCESS_MAINT_TUNNELS)
 
+/datum/job/proc/get_card_trim()
+	return src.card_trim
```

Comment:
getter procs are common in other languages, but I'm not sure how useful they are here, though, since we don't have private variables.
I'd just access the variable directly and avoid the proc overhead, plus the temptation of future coders to redefine this proc.